### DISCUSSION TITLE
Why example with-redux create new instance in devtools?

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [vercel/next.js#17235] [Help]
Why example with-redux create new instance in devtools?

**Comment by popuguytheparrot** [comment]
up

**Comment by julien-amblard** [comment]
I've also find out this weird behaviour, I think it's because at each `getServerSidePropd`/`getStaticProps` it create a new store with `initialReduxState`, I'm still lookig for a solution to this.

If someone have an idea I'll take it, and i'll call him "my hero"
